### ISSUE TITLE
jdk22 OpenJDK jdk_vector_byte64_j9 passed but failed

### ISSUE THREAD

**Issue by pshipton** [eclipse-openj9/openj9#18704]
https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_aarch64_mac_Personal_testList_2/1
jdk_vector_byte64_j9_0
```
16:16:59  #INFO:  AOT header validation failed: AsyncCompilation feature mismatch.
16:16:59  
16:16:59  #INFO:  StartTime: Jan 06 08:16:59 2024
16:16:59  #INFO:  Free Physical Memory: 6362 MB estimated
16:16:59  #INFO:  CPU entitlement = 800.00
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 0000000120822300, thread ID 1
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 000000012083EB00, thread ID 4
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 0000000120847F00, thread ID 5
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 0000000120850100, thread ID 6
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 0000000120835700, thread ID 3
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 000000012082D500, thread ID 2
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 000000012008A900, thread ID 0
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 0000000120859500, thread ID 7
16:17:00  XML output with verification to /Users/jenkins/workspace/Test_openjdk22_j9_sanity.openjdk_aarch64_mac_Personal_testList_2/aqa-tests/TKG/output_17044885757263/jdk_vector_byte64_j9_0/work
16:17:00  #INFO:  AOT header validation failed: AsyncCompilation feature mismatch.
16:17:00  
16:17:00  #INFO:  StartTime: Jan 06 08:16:59 2024
16:17:00  #INFO:  Free Physical Memory: 6359 MB estimated
16:17:00  #INFO:  CPU entitlement = 800.00
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 000000010F827500, thread ID 2
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 0000000130098700, thread ID 4
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 000000013008F300, thread ID 0
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 00000001300A1B00, thread ID 5
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 000000010F82F700, thread ID 3
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 000000010F81C300, thread ID 1
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 00000001300A9D00, thread ID 6
16:17:00  #INFO:  Stopping compilation thread, vmThread pointer 00000001300B3100, thread ID 7
16:19:06  Test results: passed: 1
16:19:06  Report written to /Users/jenkins/workspace/Test_openjdk22_j9_sanity.openjdk_aarch64_mac_Personal_testList_2/aqa-tests/TKG/output_17044885757263/jdk_vector_byte64_j9_0/report/html/report.html
16:19:06  Results written to /Users/jenkins/workspace/Test_openjdk22_j9_sanity.openjdk_aarch64_mac_Personal_testList_2/aqa-tests/TKG/output_17044885757263/jdk_vector_byte64_j9_0/work
16:19:06  grep #VECTOR API /Users/jenkins/workspace/Test_openjdk22_j9_sanity.openjdk_aarch64_mac_Personal_testList_2/aqa-tests/TKG/../TKG/output_17044885757263/jdk_vector_byte64_j9_0/work/jdk/incubator/vector/Byte64VectorTests.jtr
16:19:06  -----------------------------------
16:19:06  jdk_vector_byte64_j9_0_FAILED
```

Possibly related to https://github.com/eclipse-openj9/openj9/issues/17185

**Comment by pshipton** [comment]
@gita-omr

**Comment by pshipton** [comment]
https://openjdk.org/jeps/460 is still an incubator so I won't add this to the jdk22 milestone.

**Comment by Akira1Saitoh** [comment]
In the above test job, other vector tests, for example `jdk_vector_int128_j9_0`, passed but only 2 operations were reported as vectorized. There are many more operations that should be vectorized.
```
[2024-01-05T21:15:14.738Z] grep #VECTOR API /Users/jenkins/workspace/Test_openjdk22_j9_sanity.openjdk_aarch64_mac_Personal_testList_2/aqa-tests/TKG/../TKG/output_17044885757263/jdk_vector_int128_j9_0/work/jdk/incubator/vector/Int128VectorTests.jtr
[2024-01-05T21:15:14.738Z] #VECTOR API: Vectorized using mLongBitsToMaskVector128Int32 in Int128VectorTests.maskFromToLongInt128VectorTestsSmokeTest(J)V at scorching
[2024-01-05T21:15:14.738Z] #VECTOR API: Vectorized using mToLongBitsVector128Int32 in Int128VectorTests.maskFromToLongInt128VectorTestsSmokeTest(J)V at scorching
[2024-01-05T21:15:14.738Z] -----------------------------------
[2024-01-05T21:15:14.738Z] jdk_vector_int128_j9_0_PASSED
[2024-01-05T21:15:14.738Z] -----------------------------------
```

**Comment by Akira1Saitoh** [comment]
I could reproduce the failure with aarch64 linux. It is not an amac specific problem.
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/37206/

**Comment by Akira1Saitoh** [comment]
I tried a simple program with the above aarch64 linux VM. Surprisingly, vectorization did not happen. It gets vectorized when [JDK21 VM](https://openj9-artifactory.osuosl.org/artifactory/ci-openj9/Build_JDK21_aarch64_linux_Nightly/144/) is used.
```
       //  ADD 
       for (int i = 0; i < a.length; i += SPECIES.length()) {
          IntVector va = IntVector.fromArray(SPECIES, a, i);
          IntVector vb = IntVector.fromArray(SPECIES, b, i);
          IntVector vc = va.add(vb);
          vc.intoArray(c, i);
       }
```

I attach the trace log below. @gita-omr can you please take a look?
[testVAdd.trc.zip](https://github.com/eclipse-openj9/openj9/files/13870373/testVAdd.trc.zip)

**Comment by pshipton** [comment]
The failures occurs across platforms and occurs too frequently to keep running in the builds. Excluding vector tests via
https://github.com/adoptium/aqa-tests/pull/5041

https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_aarch64_linux_Nightly_testList_1/4/
jdk_vector_float128_j9_0 
https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_aarch64_linux_Nightly_testList_2/4/
jdk_vector_double128_j9_0
jdk_vector_byte64_j9_0

https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_ppc64_aix_Nightly_testList_0/1
jdk_vector_int128_j9_0
https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_ppc64_aix_Nightly_testList_1/1
jdk_vector_double128_j9_0 
jdk_vector_long128_j9_0 
jdk_vector_byte64_j9_0 
https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_ppc64_aix_Nightly_testList_2/1
jdk_vector_float128_j9_0 
jdk_vector_short128_j9_0 
jdk_vector_byte128_j9_0

https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_ppc64le_linux_Nightly_testList_0/1
jdk_vector_int128_j9_0
https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_ppc64le_linux_Nightly_testList_1/1
jdk_vector_double128_j9_0 
jdk_vector_long128_j9_0 
jdk_vector_byte64_j9_0
https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_ppc64le_linux_Nightly_testList_2/1
jdk_vector_float128_j9_0 
jdk_vector_short128_j9_0 
jdk_vector_byte128_j9_0

https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_s390x_linux_Nightly_testList_0/1
jdk_vector_int128_j9_0
https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_s390x_linux_Nightly_testList_1/1
jdk_vector_double128_j9_0 
jdk_vector_long128_j9_0 
jdk_vector_byte64_j9_0
https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_s390x_linux_Nightly_testList_2/1
jdk_vector_float128_j9_0 
jdk_vector_short128_j9_0 
jdk_vector_byte128_j9_0

https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_x86-64_linux_Nightly_testList_0/1/
jdk_vector_int128_j9_0
https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_x86-64_linux_Nightly_testList_1/1
jdk_vector_double128_j9_0 
jdk_vector_long128_j9_0 
jdk_vector_byte64_j9_0
https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_x86-64_linux_Nightly_testList_2/1
jdk_vector_float128_j9_0 
jdk_vector_short128_j9_0 
jdk_vector_byte128_j9_0

https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_x86-64_mac_Nightly_testList_0/1
jdk_vector_int128_j9_0
https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_x86-64_mac_Nightly_testList_1/1
jdk_vector_double128_j9_0
jdk_vector_long128_j9_0
jdk_vector_byte64_j9_0
https://openj9-jenkins.osuosl.org/job/Test_openjdk22_j9_sanity.openjdk_x86-64_mac_Nightly_testList_2/1
jdk_vector_float128_j9_0 
jdk_vector_short128_j9_0 
jdk_vector_byte128_j9_0

**Comment by gita-omr** [comment]
This PR should fix the problem: https://github.com/eclipse-openj9/openj9/pull/18965

**Comment by pshipton** [comment]
https://github.com/eclipse-openj9/openj9/pull/19192 is merged to 0.44.

**Comment by pshipton** [comment]
Created https://github.com/adoptium/aqa-tests/pull/5160 to re-include the tests.
